ui-grid/ui-grid.d.ts

```diff
@@ -604,7 +664,7 @@ declare module uiGrid {
         leaveOpen?: boolean;
     }
     export interface ISortInfo {
-        direction?: number;
+        direction?: string;
```

Comment:
This should be uiGridConstants.ASC or uiGridConstants.DESC which are both numbers